manticore/core/cpu/x86.py

```diff
@@ -5636,99 +5620,46 @@ def VZEROUPPER(cpu):
 ################################################################################
 #Calling conventions
 
-class ABI:
-    '''IA32 Calling conventions
-        https://en.wikipedia.org/wiki/X86_calling_conventions
+class AMD64ABI(ABI):
```

Comment:
I don't think `AMD64ABI` is quite accurate, i feel like this should be `SystemV(ABI)`. the Microsoft x64 ABI is different, for example: https://en.wikipedia.org/wiki/X86_calling_conventions#x86-64_calling_conventions.